features/adres.feature

```diff
@@ -20,22 +24,22 @@ Functionaliteit: Adresvelden vullen
       | burgerservicenummer | <burgerservicenummer>                        |
       | fields              | verblijfplaats                               |
       Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
-      | naam                      | waarde      |
-      | type                      | Adres       |
-      | functieAdres.code         | W           |
-      | functieAdres.omschrijving | woonadres   |
-      | verblijfadres.straat      | <straat>    |
-      | verblijfadres.korteNaam   | <korteNaam> |
+      | naam                              | waarde               |
+      | type                              | Adres                |
+      | functieAdres.code                 | W                    |
+      | functieAdres.omschrijving         | woonadres            |
+      | verblijfadres.officieleStraatnaam | <officielestraat>    |
+      | verblijfadres.korteStraatNaam     | <korteNaam>          |
 
       Voorbeelden:
```

Comment:
Waarom heb  je hier aparte kolommen opgenomen voor 'officielestraat' en 'korteNaam'? De waarden daarin zijn immers gelijk aan de waarden in de kolommen 'naam openbare ruimte' en 'straatnaam'.

Reply:
Ik heb niets toegevoegd. Ik heb alleen de namen gewijzigd conform afspraak.

Reply:
maar is door de wijziging wel irrelevant geworden, dus ik denk dat de opmerking van Robert terecht is dat de kolommen van voorbeeld ontdubbeld kunnen worden

Reply:
Ik ga er vanuit dat je dit meeneemt. Als je wil dat ik er een issue voor opvoer dan hoor ik dat graag.

Reply:
Dit scenario is verwijderd.